### ISSUE TITLE
[External NIP05] Editing or deleting an external NIP05 ID

### ISSUE THREAD

**Issue by setch-l** [planetary-social/nos#901]
Given that I am a user editing / deleting my external NIP05 ID
I should see a warning message that says
Note: This change also needs to be updated with your NIP05 ID provider.

**Comment by rabble** [comment]
Shouldn't we use less technical language. "You've now setup user@nos.social as your name on the nostr network" because users aren't really needing to know what an npub or nip05 are.

**Comment by Chardot** [comment]
> Shouldn't we use less technical language. "You've now setup [user@nos.social](mailto:user@nos.social) as your name on the nostr network" because users aren't really needing to know what an npub or nip05 are.

I agree, but note that this case is for existing Nostr users who have registered a NIP-05 in a third party server and associated it with their NPUB. So this specific group of users definitely know what a NIP-05 is.

**Comment by Chardot** [comment]
Here's a preview of the designs:

![Image](https://github.com/planetary-social/nos/assets/477835/dde28cdf-bd1c-47b2-8d1f-0f27a5af47f0)

See [this design](https://www.figma.com/file/s0qf4VmyQygydP8MIQazZc/Nos?type=design&node-id=4978%3A4156&mode=design&t=DSWXwfFegirdlMJQ-1) and the [rest of the user flows](https://www.figma.com/file/s0qf4VmyQygydP8MIQazZc/Nos?type=design&node-id=3996-216&mode=design) in Figma
